src/routes/video.route.js

```diff
@@ -0,0 +1,24 @@
+const M = require('../models');
+const logger = require('../helpers/logger');
+
+module.exports = (router) => {
+  router.post('/video', async (req, res) => {
+    logger.log('info', 'received request: POST /video\nbody:', req.body);
+    M.Video.create(req.body)
+      .then((video) => res.status(200).send(video))
+      .catch((err) => {
+        logger.log('error', 'POST /video failed', err);
+        res.sendStatus(500);
+      });
+  });
+
+  router.get('/video/getByName', async (req, res) => {
```

Comment:
you should pass parameters for a get request like so: https://expressjs.com/en/guide/routing.html#route-parameters